x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -32,6 +32,7 @@ const enabledActionTypes = [
   'test.index-record',
```

Comment:
Remainder of the files are basic tests for event log usage by actions and alerts.  They weren't added when event log support was added to actions and alerts, because we didn't have the find function built yet.  These tests would have caught the bugs fixed in this PR.